src/auth/auth.service.ts

```
@@ -15,73 +16,93 @@ export class AuthService {
     private userService: UserService,
     private jwtService: JwtService,
     private prisma: PrismaService,
+    private readonly logger: WinstonLoggerService,
   ) {}
 
   async validateUser(username: string, password: string): Promise<any> {
+    this.logger.log(`Validating user: ${username}`);
     const user = await this.userService.getByUsername(username);
 
     if (!user) {
+      this.logger.warn(`User not found: ${username}`);
       return null;
     }
 
     if (!user.password) {
+      this.logger.error(`Password is missing for user: ${username}`);
```

Throw exeption instead of log